src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -324,7 +324,7 @@
  *    <tr>
  *       <th scope="row" style="font-weight:normal">{@code unit}
  *       <td>{@link ListFormat#getInstance(Locale, ListFormat.Type, ListFormat.Style)
- *       ListFormat.getInstance}{@code (getLocale()}, {@link ListFormat.Type#UNIT}, {@link ListFormat.Style#FULL}}
+ *       ListFormat.getInstance}{@code (getLocale()}, {@link ListFormat.Type#UNIT}, {@link ListFormat.Style#FULL})
```

Comment:
The new ')' at the end of the line.  Where's the matching '('?  I'm guessing that the IDE thinks it's the one immediately before the `getLocale()`, but I suspect that javadoc won't agree with that.

Reply:
> Where's the matching `(`?  I'm guessing that the IDE thinks it's the one immediately before the `getLocale()`, but I suspect that javadoc won't agree with that.

You are correct - the matching `(` is the one preceding `getLocale()`.

I'm not sure what you mean by "I suspect that javadoc won't agree with that". This change is simply making this row of the table consistent with the ones above it. You can see how they currently appear [here](https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/text/MessageFormat.html). In this row, the closing `}` should have been a `)`.

Thanks for taking a look.

Reply:
Shouldn't this closing `)` be wrapped in `{@code}` like `{@code )}` to match with the code format in `{@code (getLocale()}`?

Reply:
> Shouldn't this closing `)` be wrapped in `{@code}` like `{@code )}` to match with the code format in `{@code (getLocale()}`?

Yes it should - thanks. The two rows above also have the same problem, i.e., I was just making this row consistent with the ones above it, but you're right that we should be fixing the font mismatch instead of propagating it.

I've fixed all three in dde1c86740d.